src/Tests/E-Games.Tests/UserTests.cs

```diff
@@ -0,0 +1,179 @@
+﻿using E_Games.Data.Data.Models;
+using E_Games.Web.Controllers;
+using E_Games.Web.ViewModels;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+using Moq;
+using System.Security.Claims;
+
+namespace E_Games.Tests
+{
+    public class UserTests
+    {
+        private readonly Mock<UserManager<ApplicationUser>> _userManagerMock;
+        private readonly UserController _controller;
```

Comment:
Your tests involve controllers. This marks, that you have your business logic spread across multiple sources. Ideally, unit test should cover only BL layer, which is, in your case, UserManager.

Reply:
You need to remove controllers from tests. You test should cover services instead. Don't assert Http results, assert actual model results